src/tests/utils/Load.test.js

```diff
@@ -17,7 +17,7 @@ describe('#unzipDependencies', () => {
   });
 
   it('should make an http request and extract data from the resulting zip file', async () => {
-    const scope = nock('https://packages.fhir.org')
+    const scope = nock('http://packages.fhir.org')
```

Comment:
So what's this all about?

Reply:
This is what happened when I mocked `https` in the `nodePolyfills` plugin. I really can't explain why things broke or why this fixed it.